charts/gatekeeper/values.yaml

```diff
@@ -16,10 +16,19 @@ affinity: {}
 tolerations: []
 podAnnotations:
   { container.seccomp.security.alpha.kubernetes.io/manager: runtime/default }
-resources:
-  limits:
-    cpu: 1000m
-    memory: 512Mi
-  requests:
-    cpu: 100m
-    memory: 256Mi
+controllerManager:
```

Comment:
This will be generated, no need to modify here. It'll be part of next release

Reply:
Shuld I rollback these changes and then run `make manifests` after that?